config/rbac/authorino_editor_role.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit authorinos.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: authorino-editor-role
```

Comment:
This will collide with the name of Authorino's `editor-role` after https://github.com/Kuadrant/authorino/pull/191 was merged. See https://github.com/Kuadrant/authorino/blob/05f151cfc1bcdf4bcd4cfc95efc367ead80df1dc/install/manifests.yaml#L1110

Maybe we should fix it in https://github.com/kuadrant/authorino, i.e. change the name of the role there to `authconfig-editor-role` (`authorino-authconfig-editor-role` after prefixed)

Reply:
Done in https://github.com/Kuadrant/authorino/pull/192.